.vscode/settings.json

```
@@ -22,5 +22,23 @@
         "."
     ],
     "python.testing.unittestEnabled": false,
-    "python.testing.pytestEnabled": true
+    "python.testing.pytestEnabled": true,
+    "files.exclude": {
+        "**/.git": true,
+        "**/.svn": true,
+        "**/.hg": true,
+        "**/CVS": true,
+        "**/.DS_Store": true,
+        "**/Thumbs.db": true,
+        ".pixi": true,
+        ".mypy_cache": true,
+        ".pytest_cache": true,
+        ".ruff_cache": true,
+        "juliaup": true
+    },
```

Would this also work? Then we don't need to keep maintaining two lists. `files.exclude` points to this setting.


```suggestion
    "explorer.excludeGitIgnore": true,
```

---

As discussed with @deltamarnix, I would miss those folders.
Maybe the setting that @visr suggested should just be added globally by people who prefer that?

---

Reverted